src/Components/Facility/ConsultationForm.tsx

```diff
@@ -552,8 +552,7 @@ export const ConsultationForm = (props: any) => {
   };
 
   const handleDoctorSelect = (doctor: UserModel | null) => {
-    doctor &&
-      doctor.id &&
+    if (doctor && doctor.id) {
```

Comment:
Use optional chaining
```suggestion
    if (doctor?.id) {
```